new/resource_metadata.md

```diff
@@ -0,0 +1,122 @@
+---
+RFC: unassigned
+Title: Resource Metadata
+Author: Thom May <thom@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Resource Metadata
+
+It's now possible to extract detailed API documentation of individual
+resources, but running other people's ruby code on your system to do so
+is unsafe. Therefore, a cookbook's metadata should contain the details of
+the resources contained in the cookbook.
+
+## Motivation
+
+    As a user of Supermarket,
+    I want to view details of resources,
```

Comment:
Would this be displayed similarly to how the Terraform registry does it? 👍

Reply:
The Inputs and Outputs sections there? Yea, that's along the lines we are picturing. With the resource documentation in the metadata.json, Supermarket could be updated to include transforming that into web content associated with a particular version of a cookbook during the upload processing.

Reply:
That's exactly what I was thinking. It's really useful 👍